Assets/MRTK/StandardAssets/Shaders/MixedRealityStandard.shader

```diff
@@ -316,6 +316,8 @@ Shader "Mixed Reality Toolkit/Standard"
 #endif
             };
 
+            CBUFFER_START(UnityPerMaterial)
```

Comment:
With the latest changes I see this shader reporting:

![image](https://user-images.githubusercontent.com/13305729/95627216-a487c000-0a30-11eb-8832-5a3518d7fc4e.png)

That might be due to some of the properties being conditionally defined in/out. We may have to remove those preprocessor definitions. ie:

`#if defined(_HOVER_LIGHT) || defined(_PROXIMITY_LIGHT) || defined(_BORDER_LIGHT)`
            `fixed _FluentLightIntensity;`
`#endif`

becomes

`fixed _FluentLightIntensity;`